test/babelish/test_csv2android.rb

```diff
@@ -13,7 +14,8 @@ def test_converting_csv_to_xml
 
   def test_converting_csv_with_special_chars
     csv_file = "test/data/android_special_chars.csv"
-    converter = Babelish::CSV2Android.new(csv_file, "german" => "de")
+    converter = Babelish::CSV2Android.new(csv_file,
+										  "german" => "de")
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Tab detected.